src/plugins/lanplus/lanplus.c

```diff
@@ -2163,25 +2161,22 @@ ipmi_lanplus_send_payload(
 					fprintf(stderr, "\n\n");
 				}
 
-
 				/*
 				 * If we are presession, and the command is GET CHANNEL AUTHENTICATION
 				 * CAPABILITIES, we will build the command in v1.5 format.  This is so
 				 * that we can ask any server whether it supports IPMI v2 / RMCP+
 				 * before we attempt to open a v2.x session.
 				 */
 				if ((ipmi_request->msg.netfn == IPMI_NETFN_APP) &&
-					 (ipmi_request->msg.cmd   == IPMI_GET_CHANNEL_AUTH_CAP) &&
-					 (session->v2_data.bmc_id  == 0)) // jme - check
+				    (ipmi_request->msg.cmd == IPMI_GET_CHANNEL_AUTH_CAP) &&
+				    (session->v2_data.bmc_id == 0)) // jme - check
 				{
-					lprintf(LOG_DEBUG+1, "BUILDING A v1.5 COMMAND");
+					lprintf(LOG_DEBUG + 1, "BUILDING A v1.5 COMMAND");
 					entry = ipmi_lanplus_build_v15_ipmi_cmd(intf, ipmi_request);
-				}
-				else
-				{
-					int isRetry = ( try > 0 ? 1 : 0 );
+				} else {
+					int isRetry = (try > 0) ? 1 : 0;
```

Comment:
This was a programming pleonasm. Once you're touching it, please keep just `(try > 0)`.
It woul also be nice to change isRetry to `bool` with `#include <stdbool.h>`.

Reply:
Ack